packages/workflow/package.json

```diff
@@ -39,6 +39,7 @@
     "@types/xml2js": "catalog:"
   },
   "dependencies": {
+    "@n8n/permissions": "workspace:*",
```

Comment:
This change is breaking some CI workflows now because we did not update a few other places that need to be updated when we add new dependencies like this.
we really need to start breaking up the workflow package instead of continuing to keep making the dependency tree more convoluted.

Reply:
I feel like some kind of shared frontend/backend types packages would go a long way.

Reply:
agreed